mentorApp/README.md

```diff
@@ -1,16 +0,0 @@
-# MentorApp
```

Comment:
Deleted this README file within `mentorApp` since we already have a README file in the home directory.